docs/QUICK_START.md

```diff
@@ -0,0 +1,202 @@
+- If you do this, you will need to flash the "siktelem" version of the firmware for your MatekSys Tx module.
+- If you solder a cable to these pads, you must then also use the soldered USB cable for flashing and not connect anything to the built-in type C jack.
+
+## 8. Radio calibration
+
+There are many calibration and setup steps required for ArduPilot on a new aircraft.  Here, we mention just the RC calibration step.  Not only because RC calibration needs to be repeated when switching to mLRS from a different RC system, but also because of a quirk you may encounter.
+
+- If you normally use QGroundControl as your ground station, be aware that it does not properly handle Radio calibration when using "rc_override".
+- If you did not connect sbus or CRSF RC output from your receiver to your flight controller, you should use another ground station such as Mission Planner or APM Planner 2.0 to perform radio calibration.
+- You can, of course, connect either of these ground stations via your mLRS Tx module or directly via USB to your flight controller.
+- This is a good time to configure and check the operation of any flight modes and switches you might want.
+
+## 9. Bench and range testing
+
+Before your first flight with mLRS, it is good to do some additional testing to ensure everything is working as expected.
+
+- Remove props and power everything up.
+- Verify the Rx and Tx modules connect by observing the LEDs.
+- Use the mLRS Lua script or CLI to set the Rx and Tx power to the lowest setting.
+- Start your ground station and verify that parameters download in a reasonable time depending on mode (6-60 seconds).
+- Verify that everything you expect such as attitude, GPS position, battery voltage, etc is being displayed correctly by both the ground station and Yaapu telemetry if you use it.  If you don't use Yaapu or don't see what you expect, check that you see the expected telemetry sensors on your radio (MDL -> TELEMETRY -> Discover new sensors)
+- Ensure you can change flight modes and arm/disarm the aircraft from both your RC radio and your ground station.
+- Perform a "walking range test" by leaving the aircraft behind while you walk away with the transmitter and ground station.
+- If mLRS doesn't work as expected, see the [Troubleshooting](TROUBLE.md) section.
+- Don't forget to set the Rx and Tx power back to a reasonable level before your first flight.
+
+## 10. Fail-safe configuration
+
+Check and test your failsafe behavior and configuration ***before*** you experience a connection loss in flight.
+
+- You may choose to configure ArduPilot for failsafe actions for loss of either MAVLink telemetry or radio control or both.
+- Note that even though both MAVLink and RC are transported by mLRS in the same radio frames, the design is such that loss of radio control may occur at a slightly longer range than loss of telemetry at the receiver.
```

Comment:
"... control occurs at a ..."

all data i have show that's always true ... for the "protected" channels only though ...

so the sentence is not totally correct, I like correct statemenets, but am not sure if that complexity isn't overwhelming here, I feel however that some recommendation to use certain channels for critical rc data might be in place.

Reply:
> all data i have show that's always true ... for the "protected" channels only though ...

I'll change may to "will probably".  I don't want to promise anything.

Here, the context is failsafe and I didn't say anything about channels.   Would it make it be more clear if I say "RC failsafe" rather than "loss of radio control"?